packages/overlays/src/OverlayController.js

```diff
@@ -423,76 +424,60 @@ export class OverlayController {
    * animation is played.
    */
   _handleBackdrop({ animation = true, phase }) {
-    if (this.placementMode === 'local') {
-      switch (phase) {
-        case 'init':
-          if (!this.backdropNode) {
-            this.backdropNode = document.createElement('div');
-            this.backdropNode.classList.add('local-overlays__backdrop');
-          }
-          this.backdropNode.slot = '_overlay-shadow-outlet';
-          this._contentNodeWrapper.parentElement.insertBefore(
-            this.backdropNode,
-            this._contentNodeWrapper,
-          );
-          break;
-        case 'show':
-          this.__hasActiveBackdrop = true;
-          break;
-        case 'hide':
-          if (!this.backdropNode) {
-            return;
-          }
-          this.__hasActiveBackdrop = false;
-          break;
-        case 'teardown':
-          if (!this.backdropNode) {
-            return;
-          }
-          this.backdropNode.parentNode.removeChild(this.backdropNode);
-          break;
-        /* no default */
-      }
-      return;
-    }
     const { backdropNode } = this;
     switch (phase) {
       case 'init':
-        this.backdropNode = document.createElement('div');
-        this.backdropNode.classList.add('global-overlays__backdrop');
+        // For global overlays, styles for this are added to head by the OverlayManager
+        // For local, we add it locally in DOM here
+        if (this.placementMode === 'local') {
+          const styleTag = document.createElement('style');
+          styleTag.setAttribute('data-local-overlays', '');
+          styleTag.textContent = localOverlaysStyle.cssText;
+          this._contentNodeWrapper.parentElement.insertBefore(styleTag, this._contentNodeWrapper);
```

Comment:
Can we assume here that we add this in shadow dom? Even if so, not conflicting with your host is not 100% guaruanteed.
If a dev purely uses OverlayController, this would add the same style tag multiple times. (think of a page full with tooltips or popovers).

Reply:
I'm fine with going with `<lion-overlays-backdrop></lion-overlays-backdrop>` and encapsulating the styles, but then I think we should do that for both local and global placement backdrops right? And if we do that, it's a somewhat substantial breaking change if people were relying on being able to style the backdrop from outside for modal dialogs for example.

Maybe an intermediate solution before reworking the overlay system to use ShadowDOM more, would be to move this style tag logic inside the `if (!this.backdrop) {} ` block, that way the style tag does not get added if the user passes their own backdrop node, meaning they are then fully able to use their own backdrop without any leftover style tags they don't need/want. Thoughts?